sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/OneCollectorChannelListener.java

```diff
@@ -9,13 +9,51 @@
  */
 public class OneCollectorChannelListener implements Channel.Listener {
 
+    /**
+     * Maximum time interval in milliseconds after which a synchronize will be triggered, regardless of queue size.
+     */
+    static final int ONE_COLLECTOR_TRIGGER_INTERVAL = 3 * 1000;
```

Comment:
Should add @VisibleForTesting to all the fields if the package private visibility is used only for testing.